packages/block-editor/src/components/spacing-sizes-control/utils.js

```diff
@@ -211,5 +206,5 @@ export function isValuesDefined( values ) {
 	if ( values === undefined || values === null ) {
 		return false;
 	}
-	return ! isEmpty( Object.values( values ).filter( ( value ) => !! value ) );
+	return Object.values( values ).filter( ( value ) => !! value ).length > 0;
```

Comment:
`Object.values()` will always return an array, so checking `.length` here will do the job.